clcache.py

```diff
@@ -1334,7 +1334,7 @@ def processDirect(cache, outputFile, compiler, cmdLine, sourceFile):
             origCmdLine = cmdLine
```

Comment:
I don't think so: we don't modify either list. We do however make `cmdLine` reference a *new* list, but doing so will not change the list referenced by `origCmdLine`. Or am I missing something? It's getting late...